<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentsWindow</class>
 <widget class="QMainWindow" name="StudentsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableView" name="studentsTableView">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="newStudentAction"/>
    <addaction name="importStudentsAction"/>
    <addaction name="separator"/>
    <addaction name="closeAction"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="newStudentAction">
   <property name="text">
    <string>Add Student ...</string>
   </property>
  </action>
  <action name="importStudentsAction">
   <property name="text">
    <string>Import Students</string>
   </property>
  </action>
  <action name="closeAction">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
